Assets/Scripts/Editor/SaveExplorerWindow.cs

```diff
@@ -528,7 +530,7 @@ bool IsReady()
                                 }
 
                                 if (unique)
-                                    saveNames[j].text = replaceText;
+                                    otherName.text = replaceText;
```

Comment:
Are you sure you're modifying saveNames here, with that change?
I'm not convinced by the usefulness of this commit

Reply:
Hi. You might be right, it's nothing important really.

As to your first question: `GUIContent` is a class and `text` is a `string text { get; set; }` property

Reply:
I see, so it works because it calls a method of the referenced object, not modify the reference